guide/slash-commands/autocomplete.md

```diff
@@ -98,12 +117,11 @@ const boolean = interaction.options.getBoolean('choice');
 const number = interaction.options.getNumber('num');
 ```
 
-However, the `.getUser()`, `.getMember()`, `.getRole()`, `.getChannel()`, `.getMentionable()` and `.getAttachment()` methods will not function the same, as Discord does not send their respective full objects until the slash command is completed. For these, you can get the Snowflake value using `interaction.options.get('option').value`:
-
+However, the `.getUser()`, `.getMember()`, `.getRole()`, `.getChannel()`, `.getMentionable()` and `.getAttachment()` methods are not available to autocomplete interactions. Discord does not send the respective full objects for these methods until the slash command is completed. For these, you can get the Snowflake value using `interaction.options.get('option').value`:
 
 ## Notes
 
-- You have to respond to the request within 3 seconds, as with other application command interactions.
-- You cannot defer the response to an autocomplete interaction.
-- After the user selects a value and sends the command, it will be received as a <DocsLink path="class/ChatInputCommandInteraction"/> with the chosen value.
-- You can only respond with a maximum of 25 choices at a time.
+- As with other application command interactions, autocomplete interactions must receive a response within 3 seconds. 
+- You cannot defer the response to an autocomplete interaction. If you're dealing with asynchronous suggestions, such as from an API, consider keeping a local cache.
+- After the user selects a value and sends the command, it will be received as a normal <DocsLink path="class/ChatInputCommandInteraction"/> with the chosen value.
```

Comment:
```suggestion
- After the user selects a value and sends the command, it will be received as a regular <DocsLink path="class/ChatInputCommandInteraction"/> with the chosen value.
```